lmp/tokenizer/_base_list_tokenizer.py

```diff
@@ -356,7 +405,19 @@ def build_vocab(
             min_count:
                 Minimum of token's frequence. If token's frequence is smaller
                 than `min_count`, then discard that token.
+
+        Raises:
+            TypeError:
+                When `batch_sequences` is not instance of `List[str]`.
+                When `min_count` is not instance of `int`.
         """
+        # Type check.
+        if not isinstance(batch_sequences, List):
+            raise TypeError(
+                '`batch_sequences` must be instance of `List[str]`.')
```

Comment:
new line before right parenthesis

Reply:
First do `batch_sequences = list(batch_sequences)`, then use `for` loop to check each element in `batch_sequences` is an instance of `str`.